docs/spec/type_system.rst

```diff
@@ -150,8 +150,38 @@ Decimal
 ~~~~~~~
 Decimals have a given precision and scale and used to store fixed-point floats like money.
 
-The decimal type ``decimal128[P, S]`` (``P`` denoting the precision and ``S`` the scale) is not touched, since the
-exact precision and scale has a semantic meaning.
+There is a single decimal type ``decimal128[P, S]`` where ``P`` measures the total precision in digits and ``S``
+measures the scale in digits (therefore :math:`P \ge S`).
+
+>>> from decimal import Context, Decimal
```

Comment:
Is this properly rendered as code?

Reply:
I realized that all code sections are not specially marked. Is this sphinx magic or an actual bug?

Reply:
Sphinx magic. Doctests don't require a marker.

Reply:
![Screen Shot 2019-05-22 at 14 25 53](https://user-images.githubusercontent.com/1529400/58174185-926f7b00-7c9d-11e9-8546-b50c31e139ea.png)